### ISSUE TITLE
Quasar crashes with Android 7.0

### ISSUE THREAD

**Issue by jhshadi** [scakemyer/plugin.video.quasar#544]
Quasar crashes on start after updating to Android 7.0, any idea what's the issue?
Running Quasar v0.9.35 with Kodi 16.1 on a Nexus Player with Android 7.0.
attached log file.

```
00:12:01 T:18446744072826169632  NOTICE: special://profile/ is mapped to: special://masterprofile/
00:12:01 T:18446744072826169632  NOTICE: -----------------------------------------------------------------------
00:12:01 T:18446744072826169632  NOTICE: Starting Kodi (16.1 Git:2016-04-24-eaa012d). Platform: Android x86 32-bit
00:12:01 T:18446744072826169632  NOTICE: Using Release Kodi x32 build
00:12:01 T:18446744072826169632  NOTICE: Kodi compiled Apr 24 2016 by GCC 4.8.0 for Android x86 32-bit API level 17 (API level 17)
00:12:01 T:18446744072826169632  NOTICE: Running on Asus Nexus Player with Android 7.0.0 API level 24, kernel: Linux x86 32-bit version 3.10.20-gb02fee9
00:12:01 T:18446744072826169632  NOTICE: FFmpeg version: 2.8.6-kodi-2.8.6-Jarvis-16.0
00:12:01 T:18446744072826169632  NOTICE: Host CPU: Intel(R) Atom(TM) CPU Z3560 @ 1.00GHz, 4 cores available
00:12:01 T:18446744072826169632  NOTICE: Product: fugu, Device: fugu, Board: fugu - Manufacturer: Asus, Brand: google, Model: Nexus Player, Hardware: fugu
00:12:01 T:18446744072826169632  NOTICE: External storage path = /storage/emulated/0; status = ok
00:12:01 T:18446744072826169632  NOTICE: special://xbmc/ is mapped to: /data/user/0/org.xbmc.kodi/cache/apk/assets
00:12:01 T:18446744072826169632  NOTICE: special://xbmcbin/ is mapped to: /data/user/0/org.xbmc.kodi/cache/apk/assets
00:12:01 T:18446744072826169632  NOTICE: special://masterprofile/ is mapped to: /storage/emulated/0/Android/data/org.xbmc.kodi/files/.kodi/userdata
00:12:01 T:18446744072826169632  NOTICE: special://home/ is mapped to: /storage/emulated/0/Android/data/org.xbmc.kodi/files/.kodi
00:12:01 T:18446744072826169632  NOTICE: special://temp/ is mapped to: /storage/emulated/0/Android/data/org.xbmc.kodi/files/.kodi/temp
00:12:01 T:18446744072826169632  NOTICE: The executable running is: /system/bin/app_process32
00:12:01 T:18446744072826169632  NOTICE: Local hostname: localhost
00:12:01 T:18446744072826169632  NOTICE: Log File is located: /storage/emulated/0/Android/data/org.xbmc.kodi/files/.kodi/temp/kodi.log
00:12:01 T:18446744072826169632  NOTICE: -----------------------------------------------------------------------
00:12:01 T:18446744072826169632  NOTICE: load settings...
00:12:01 T:18446744072826169632   DEBUG: CSettings: loaded settings definition from special://xbmc/system/settings/settings.xml
00:12:01 T:18446744072826169632 WARNING: CSettingInt: unknown options filler "audiocdactions" of "audiocds.autoaction"
00:12:01 T:18446744072826169632   DEBUG: CSettings: loaded settings definition from special://xbmc/system/settings/android.xml
00:12:01 T:18446744072826169632   DEBUG: AESinkAUDIOTRACK - 44100 supported
00:12:01 T:18446744072826169632   DEBUG: AESinkAUDIOTRACK - 48000 supported
00:12:01 T:18446744072826169632   DEBUG: AESinkAUDIOTRACK - 96000 supported
00:12:01 T:18446744072826169632   DEBUG: AESinkAUDIOTRACK - 192000 supported
00:12:01 T:18446744072826169632  NOTICE: Found 1 Lists of Devices
00:12:01 T:18446744072826169632  NOTICE: Enumerated AUDIOTRACK devices:
00:12:01 T:18446744072826169632  NOTICE:     Device 1
00:12:01 T:18446744072826169632  NOTICE:         m_deviceName      : AudioTrack
00:12:01 T:18446744072826169632  NOTICE:         m_displayName     : android
00:12:01 T:18446744072826169632  NOTICE:         m_displayNameExtra: audiotrack
00:12:01 T:18446744072826169632  NOTICE:         m_deviceType      : AE_DEVTYPE_HDMI
00:12:01 T:18446744072826169632  NOTICE:         m_channels        : FL,FR,FC,LFE,BL,BR,SL,SR
00:12:01 T:18446744072826169632  NOTICE:         m_sampleRates     : 48000,44100,48000,96000,192000
00:12:01 T:18446744072826169632  NOTICE:         m_dataFormats     : AE_FMT_S16LE,AE_FMT_AC3,AE_FMT_DTS
00:12:01 T:18446744072826169632   DEBUG: CSkinSettings: no <skinsettings> tag found
00:12:01 T:18446744072826169632  NOTICE: No settings file to load (special://xbmc/system/advancedsettings.xml)
00:12:01 T:18446744072826169632  NOTICE: No settings file to load (special://masterprofile/advancedsettings.xml)
00:12:01 T:18446744072826169632  NOTICE: Default DVD Player: dvdplayer
00:12:01 T:18446744072826169632  NOTICE: Default Video Player: dvdplayer
00:12:01 T:18446744072826169632  NOTICE: Default Audio Player: paplayer
00:12:01 T:18446744072826169632  NOTICE: Disabled debug logging due to GUI setting. Level 0.
00:12:01 T:18446744072826169632  NOTICE: Log level changed to "LOG_LEVEL_NORMAL"
00:12:01 T:18446744072826169632  NOTICE: CMediaSourceSettings: loading media sources from special://masterprofile/sources.xml
00:12:01 T:18446744072826169632  NOTICE: Loading player core factory settings from special://xbmc/system/playercorefactory.xml.
00:12:01 T:18446744072826169632  NOTICE: Loaded playercorefactory configuration
00:12:01 T:18446744072826169632  NOTICE: Loading player core factory settings from special://masterprofile/playercorefactory.xml.
00:12:01 T:18446744072826169632  NOTICE: special://masterprofile/playercorefactory.xml does not exist. Skipping.
00:12:02 T:18446744072826169632  NOTICE: Running database version Addons20
00:12:02 T:18446744072826169632  NOTICE: ADDONS: Using repository repository.exodus
00:12:02 T:18446744072826169632  NOTICE: ADDONS: Using repository repository.unofficialquasar
00:12:02 T:18446744072826169632  NOTICE: ADDONS: Using repository repository.xbmc.org
00:12:02 T:18446744072826169632  NOTICE: ADDONS: Using repository repository.tknorris.release
00:12:02 T:18446744072826169632  NOTICE: ADDONS: Using repository repository.xbmchub
00:12:02 T:18446744072826169632  NOTICE: ADDONS: Using repository repository.quasar
00:12:02 T:18446744072826169632  NOTICE: virtual bool CWinSystemEGL::InitWindowSystem(): Using EGL Implementation: android
00:12:02 T:18446744072826169632  NOTICE: CEGLNativeTypeAndroid: Switching to resolution:
00:12:02 T:18446744072826169632  NOTICE: Current resolution: 1920x1080 @ 60.00 - Full Screen
00:12:02 T:18446744072826169632  NOTICE: Previous line repeats 1 times.
00:12:02 T:18446744072826169632  NOTICE: Found resolution 1920 x 1080 for display 0 with 1920 x 1080 @ 24.000000 Hz
00:12:02 T:18446744072826169632  NOTICE: Found resolution 1920 x 1080 for display 0 with 1920 x 1080 @ 30.000000 Hz
00:12:02 T:18446744072826169632  NOTICE: Found resolution 1920 x 1080 for display 0 with 1920 x 1080 @ 50.000000 Hz
00:12:02 T:18446744072826169632  NOTICE: Found resolution 1920 x 1080 for display 0 with 1920 x 1080 @ 60.000004 Hz
00:12:02 T:18446744072826169632  NOTICE: Found (1920x1080@60.000004) at 19, setting to RES_DESKTOP at 16
00:12:02 T:18446744072826169632  NOTICE: Checking resolution 16
00:12:02 T:18446744072826169632  NOTICE: Current resolution: 1920x1080 @ 60.00 - Full Screen
00:12:02 T:18446744072826169632  NOTICE: CEGLNativeTypeAndroid: Switching to resolution: 1920x1080 @ 60.00 - Full Screen
00:12:02 T:18446744072826169632  NOTICE: GL_VENDOR = Imagination Technologies
00:12:02 T:18446744072826169632  NOTICE: GL_RENDERER = PowerVR Rogue G6430
00:12:02 T:18446744072826169632  NOTICE: GL_VERSION = OpenGL ES 3.1 build 1.6@4050431
00:12:02 T:18446744072826169632  NOTICE: GL_SHADING_LANGUAGE_VERSION = OpenGL ES GLSL ES 3.10 build 1.6@4050431
00:12:02 T:18446744072826169632  NOTICE: GL_EXTENSIONS = GL_EXT_debug_marker GL_EXT_blend_minmax GL_EXT_color_buffer_float GL_EXT_copy_image GL_EXT_discard_framebuffer GL_EXT_draw_buffers GL_EXT_draw_buffers_indexed GL_EXT_draw_elements_base_vertex GL_EXT_geometry_point_size GL_EXT_geometry_shader GL_EXT_gpu_shader5 GL_EXT_multi_draw_arrays GL_EXT_multisampled_render_to_texture GL_EXT_occlusion_query_boolean GL_EXT_primitive_bounding_box GL_EXT_pvrtc_sRGB GL_EXT_read_format_bgra GL_EXT_robustness GL_EXT_separate_shader_objects GL_EXT_shader_framebuffer_fetch GL_EXT_shader_io_blocks GL_EXT_shader_pixel_local_storage GL_EXT_shader_pixel_local_storage2 GL_EXT_shader_texture_lod GL_EXT_shadow_samplers GL_EXT_tessellation_point_size GL_EXT_tessellation_shader GL_EXT_texture_border_clamp GL_EXT_texture_buffer GL_EXT_texture_cube_map_array GL_EXT_texture_filter_anisotropic GL_EXT_texture_format_BGRA8888 GL_EXT_texture_rg GL_EXT_texture_sRGB_decode GL_EXT_texture_sRGB_R8 GL_EXT_texture_sRGB_RG8 GL_EXT_shader_group_vote GL_EXT_polygon_offset_clamp GL_IMG_multisampled_render_to_texture GL_IMG_program_binary GL_IMG_read_format GL_IMG_shader_binary GL_IMG_texture_compression_pvrtc GL_IMG_texture_compression_pvrtc2 GL_IMG_texture_filter_cubic GL_IMG_texture_format_BGRA8888 GL_IMG_texture_npot GL_IMG_framebuffer_downsample GL_KHR_blend_equation_advanced GL_KHR_blend_equation_advanced_coherent GL_KHR_debug GL_KHR_robustness GL_OES_compressed_ETC1_RGB8_texture GL_OES_depth24 GL_OES_depth_texture GL_OES_draw_elements_base_vertex GL_OES_EGL_image GL_OES_EGL_image_external GL_OES_EGL_image_external_essl3 GL_OES_EGL_sync GL_OES_element_index_uint GL_OES_fragment_precision_high GL_OES_draw_buffers_indexed GL_OES_geometry_point_size GL_OES_geometry_shader GL_OES_get_program_binary GL_OES_gpu_shader5 GL_OES_mapbuffer GL_OES_packed_depth_stencil GL_OES_required_internalformat GL_OES_rgb8_rgba8 GL_OES_sample_shading GL_OES_sample_variables GL_OES_shader_image_atomic GL_OES_shader_io_blocks GL_OES_shader_multisample_interpolation GL_OES_standard_derivatives GL_OES_surfaceless_context GL_OES_tessellation_point_size GL_OES_tessellation_shader GL_OES_texture_border_clamp GL_OES_texture_buffer GL_OES_texture_cube_map_array GL_OES_texture_float GL_OES_texture_half_float GL_OES_texture_npot GL_OES_texture_stencil8 GL_OES_texture_storage_multisample_2d_array GL_OES_vertex_array_object GL_OES_vertex_half_float
00:12:02 T:18446744072826169632  NOTICE: Current resolution: 1920x1080 @ 60.00 - Full Screen
00:12:03 T:18446744072826169632  NOTICE: Running database version Addons20
00:12:03 T:18446744072826169632  NOTICE: Running database version ViewModes6
00:12:03 T:18446744072826169632  NOTICE: Running database version Textures13
00:12:03 T:18446744072826169632  NOTICE: Running database version MyMusic56
00:12:03 T:18446744072826169632  NOTICE: Running database version MyVideos99
00:12:03 T:18446744072826169632  NOTICE: Running database version TV29
00:12:03 T:18446744072826169632  NOTICE: Running database version Epg11
00:12:03 T:18446744072826169632 WARNING: JSONRPC: Could not parse type "Setting.Details.SettingList"
00:12:03 T:18446744072826169632  NOTICE: ActiveAE DSP - starting
00:12:03 T:18446744072826169632  NOTICE: initialize done
00:12:03 T:18446744072826169632  NOTICE: Running the application...
00:12:04 T:18446744072826169632  NOTICE: starting zeroconf publishing
00:12:04 T:18446744072826169632  NOTICE: starting upnp client
00:12:04 T:18446744072739617056  NOTICE: ES: Starting UDP Event server on 0.0.0.0:9777
00:12:04 T:18446744072739617056  NOTICE: UDP: Listening on port 9777
00:12:04 T:18446744072826169632  NOTICE: Current resolution: 1920x1080 @ 60.00 - Full Screen
00:12:04 T:18446744072826169632  NOTICE: GL_VENDOR = Imagination Technologies
00:12:04 T:18446744072826169632  NOTICE: GL_RENDERER = PowerVR Rogue G6430
00:12:04 T:18446744072826169632  NOTICE: GL_VERSION = OpenGL ES 3.1 build 1.6@4050431
00:12:04 T:18446744072826169632  NOTICE: GL_SHADING_LANGUAGE_VERSION = OpenGL ES GLSL ES 3.10 build 1.6@4050431
00:12:04 T:18446744072826169632  NOTICE: GL_EXTENSIONS = GL_EXT_debug_marker GL_EXT_blend_minmax GL_EXT_color_buffer_float GL_EXT_copy_image GL_EXT_discard_framebuffer GL_EXT_draw_buffers GL_EXT_draw_buffers_indexed GL_EXT_draw_elements_base_vertex GL_EXT_geometry_point_size GL_EXT_geometry_shader GL_EXT_gpu_shader5 GL_EXT_multi_draw_arrays GL_EXT_multisampled_render_to_texture GL_EXT_occlusion_query_boolean GL_EXT_primitive_bounding_box GL_EXT_pvrtc_sRGB GL_EXT_read_format_bgra GL_EXT_robustness GL_EXT_separate_shader_objects GL_EXT_shader_framebuffer_fetch GL_EXT_shader_io_blocks GL_EXT_shader_pixel_local_storage GL_EXT_shader_pixel_local_storage2 GL_EXT_shader_texture_lod GL_EXT_shadow_samplers GL_EXT_tessellation_point_size GL_EXT_tessellation_shader GL_EXT_texture_border_clamp GL_EXT_texture_buffer GL_EXT_texture_cube_map_array GL_EXT_texture_filter_anisotropic GL_EXT_texture_format_BGRA8888 GL_EXT_texture_rg GL_EXT_texture_sRGB_decode GL_EXT_texture_sRGB_R8 GL_EXT_texture_sRGB_RG8 GL_EXT_shader_group_vote GL_EXT_polygon_offset_clamp GL_IMG_multisampled_render_to_texture GL_IMG_program_binary GL_IMG_read_format GL_IMG_shader_binary GL_IMG_texture_compression_pvrtc GL_IMG_texture_compression_pvrtc2 GL_IMG_texture_filter_cubic GL_IMG_texture_format_BGRA8888 GL_IMG_texture_npot GL_IMG_framebuffer_downsample GL_KHR_blend_equation_advanced GL_KHR_blend_equation_advanced_coherent GL_KHR_debug GL_KHR_robustness GL_OES_compressed_ETC1_RGB8_texture GL_OES_depth24 GL_OES_depth_texture GL_OES_draw_elements_base_vertex GL_OES_EGL_image GL_OES_EGL_image_external GL_OES_EGL_image_external_essl3 GL_OES_EGL_sync GL_OES_element_index_uint GL_OES_fragment_precision_high GL_OES_draw_buffers_indexed GL_OES_geometry_point_size GL_OES_geometry_shader GL_OES_get_program_binary GL_OES_gpu_shader5 GL_OES_mapbuffer GL_OES_packed_depth_stencil GL_OES_required_internalformat GL_OES_rgb8_rgba8 GL_OES_sample_shading GL_OES_sample_variables GL_OES_shader_image_atomic GL_OES_shader_io_blocks GL_OES_shader_multisample_interpolation GL_OES_standard_derivatives GL_OES_surfaceless_context GL_OES_tessellation_point_size GL_OES_tessellation_shader GL_OES_texture_border_clamp GL_OES_texture_buffer GL_OES_texture_cube_map_array GL_OES_texture_float GL_OES_texture_half_float GL_OES_texture_npot GL_OES_texture_stencil8 GL_OES_texture_storage_multisample_2d_array GL_OES_vertex_array_object GL_OES_vertex_half_float
00:12:05 T:18446744072763214112  NOTICE: 1Channel: Service: Installed Version: 2.5.72
00:12:05 T:18446744072763214112  NOTICE: 1Channel: Loading sqlite3 as DB engine
00:12:05 T:18446744072763214112  NOTICE: 1Channel: Service: Resetting...
00:12:05 T:18446744072763214112  NOTICE: 1Channel: Service: starting...
00:13:02 T:18446744072703449376  NOTICE: [plugin.video.quasar] quasard: starting quasard
00:13:02 T:18446744072704493856  NOTICE: [plugin.video.quasar] quasar: starting jsonrpc service
00:13:02 T:18446744072703449376  NOTICE: [plugin.video.quasar] Detected binary folder: /storage/emulated/0/Android/data/org.xbmc.kodi/files/.kodi/addons/plugin.video.quasar/resources/bin/android_x86
00:13:02 T:18446744072703449376  NOTICE: [plugin.video.quasar] Using binary folder: /storage/emulated/0/Android/data/org.xbmc.kodi/files/.kodi/addons/plugin.video.quasar/resources/bin/android_x86
00:13:02 T:18446744072703449376  NOTICE: [plugin.video.quasar] Updating quasar daemon...
00:13:02 T:18446744072703449376  NOTICE: [plugin.video.quasar] Deleting stale files set([])
00:13:02 T:18446744072703449376  NOTICE: [plugin.video.quasar] WARNING: linker: Warning: unable to normalize ""
00:13:02 T:18446744072703449376  NOTICE: [plugin.video.quasar] CANNOT LINK EXECUTABLE "/data/data/org.xbmc.kodi/files/plugin.video.quasar/bin/android_x86/quasar": /data/data/org.xbmc.kodi/files/plugin.video.quasar/bin/android_x86/quasar: has text relocations
00:13:03 T:18446744072703449376  NOTICE: [plugin.video.quasar] System information: android_x86 3.10.20-gb02fee9
00:13:03 T:18446744072703449376  NOTICE: [plugin.video.quasar] Kodi build version: 16.1 Git:2016-04-24-eaa012d
00:13:03 T:18446744072703449376  NOTICE: [plugin.video.quasar] OS type: Linux
00:13:03 T:18446744072703449376  NOTICE: [plugin.video.quasar] uname: ('Linux', 'localhost', '3.10.20-gb02fee9', '#1 SMP PREEMPT Wed Jul 20 21:30:04 UTC 2016', 'i686', '')
00:13:06 T:18446744072703449376  NOTICE: [plugin.video.quasar] quasard: starting quasard
00:13:06 T:18446744072703449376  NOTICE: [plugin.video.quasar] Detected binary folder: /storage/emulated/0/Android/data/org.xbmc.kodi/files/.kodi/addons/plugin.video.quasar/resources/bin/android_x86
00:13:06 T:18446744072703449376  NOTICE: [plugin.video.quasar] Using binary folder: /storage/emulated/0/Android/data/org.xbmc.kodi/files/.kodi/addons/plugin.video.quasar/resources/bin/android_x86
00:13:06 T:18446744072703449376  NOTICE: [plugin.video.quasar] Deleting stale files set([])
00:13:07 T:18446744072703449376  NOTICE: [plugin.video.quasar] System information: android_x86 3.10.20-gb02fee9
00:13:07 T:18446744072703449376  NOTICE: [plugin.video.quasar] Kodi build version: 16.1 Git:2016-04-24-eaa012d
00:13:07 T:18446744072703449376  NOTICE: [plugin.video.quasar] OS type: Linux
00:13:07 T:18446744072703449376  NOTICE: [plugin.video.quasar] uname: ('Linux', 'localhost', '3.10.20-gb02fee9', '#1 SMP PREEMPT Wed Jul 20 21:30:04 UTC 2016', 'i686', '')
00:13:10 T:18446744072703449376  NOTICE: [plugin.video.quasar] quasard: starting quasard
00:13:10 T:18446744072703449376  NOTICE: [plugin.video.quasar] Detected binary folder: /storage/emulated/0/Android/data/org.xbmc.kodi/files/.kodi/addons/plugin.video.quasar/resources/bin/android_x86
00:13:10 T:18446744072703449376  NOTICE: [plugin.video.quasar] Using binary folder: /storage/emulated/0/Android/data/org.xbmc.kodi/files/.kodi/addons/plugin.video.quasar/resources/bin/android_x86
00:13:10 T:18446744072703449376  NOTICE: [plugin.video.quasar] Deleting stale files set([])
00:13:10 T:18446744072703449376  NOTICE: [plugin.video.quasar] WARNING: linker: Warning: unable to normalize ""
00:13:10 T:18446744072703449376  NOTICE: [plugin.video.quasar] CANNOT LINK EXECUTABLE "/data/data/org.xbmc.kodi/files/plugin.video.quasar/bin/android_x86/quasar": /data/data/org.xbmc.kodi/files/plugin.video.quasar/bin/android_x86/quasar: has text relocations
00:13:10 T:18446744072703449376  NOTICE: [plugin.video.quasar] System information: android_x86 3.10.20-gb02fee9
00:13:10 T:18446744072703449376  NOTICE: [plugin.video.quasar] Kodi build version: 16.1 Git:2016-04-24-eaa012d
00:13:10 T:18446744072703449376  NOTICE: [plugin.video.quasar] OS type: Linux
00:13:10 T:18446744072703449376  NOTICE: [plugin.video.quasar] uname: ('Linux', 'localhost', '3.10.20-gb02fee9', '#1 SMP PREEMPT Wed Jul 20 21:30:04 UTC 2016', 'i686', '')
```

**Comment by sharadaj** [comment]
Having the same issue with my Nexus Player on 7.0. Will let you know if I figure out a fix.

**Comment by lcohen999** [comment]
Any luck ?

**Comment by scakemyer** [comment]
As mentioned in #524, this is related to permissions in Android 7 and I doubt there's anything we can do about it within Quasar itself. Someone with a working setup will have to post the exact steps needed to get it working.

**Comment by sask3m** [comment]
So anyone got this working on new shield tv with N?

**Comment by sharadaj** [comment]
In case anyone is interested, I ended up just downgrading my Nexus Player back to the last release of Android 6.0.1, rooting it, and using a sideloaded app called MyAndroidTools to disable updates. Quasar is working perfectly again with this setup. If anyone is looking for detailed instructions just post here and I'll put together a quick tutorial. Thanks for all your hard work, @scakemyer!

**Comment by sask3m** [comment]
I did the same the other day as well. :)    I also used Android Tools and managed to disable the system update service in Google Framework but can't seem to in Google services.   What all did you disable to stop updates? Thanks.  By the way I did see someone post that Quasar is working using the Nvidia Shield with 7.0 so that is good news if true.

**Comment by sharadaj** [comment]
Hey sorry for the late reply, I've been quite busy the past few days. I disabled the SystemUpdateService for both google play services and google framework. The app I used was this one:

https://play.google.com/store/apps/details?id=cn.wq.myandroidtools

Try with this one instead, might give you more luck!

**Comment by sharadaj** [comment]
@sask3m you doing this on a nexus player or your sheild TV?

**Comment by sask3m** [comment]
On Nexus player.  I did however root my Shield as well to keep it at 6.0 until I know for sure it will work at 7.0.  

   That is the app I used but for some reason I can't seem to disable the google services system update. When I go into google services and try to disable system update nothing seems to happen, can't get the blue bar to move to the off position.

**Comment by sharadaj** [comment]
Hmm weird. Glad it's working though lol as long as it's not updating you should be fine. :) 

You did find an apk for the most updated version of the app, yeah? 0.9.8.9? Other than that, I'm out of ideas lol.

**Comment by sask3m** [comment]
Wondering if you happen to have tried your Nexus player and Quasar with
Kodi 17?  I'm getting can't find Binary issue.  I am however able to use
Torrenter which I couldn't before.

**Comment by compyy** [comment]
Any plan to fix it, seeing following error on Shield, Android Naugat.

IOError: [Errno 21] Is a directory: '/data/data/org.xbmc.kodi/files/plugin.video.quasar/bin/android_arm/quasar'

http://paste.ubuntu.com/23989653/

I am ok to test the fix, if you guide what is the real issue here.

**Comment by sask3m** [comment]
What exactly is the issue, Quasar is working fine on my shield with either kodi or spmc.

**Comment by scakemyer** [comment]
With no crazy downgrading to Android 6.0?

**Comment by sask3m** [comment]
We had to downgrade to 6.0 using a nexus player, Nvidia Shield tv works
fine with 7.0.

**Comment by sask3m** [comment]
Using arm version of quasar and don't forget to enable both remote control
options under settings/services/control.

**Comment by scakemyer** [comment]
Wondering if bumping the API level wouldn't help here... After all we don't support Kodi 15 anymore, and I think we could to drop support for Android 4 also.

**Comment by lcohen999** [comment]
That would break a lot of crappy Android boxes still (stuck) on Android 4



*From:* scakemyer [mailto:notifications@github.com]
*Sent:* February 13, 2017 4:13 PM
*To:* scakemyer/plugin.video.quasar <plugin.video.quasar@noreply.github.com>
*Cc:* lcohen999 <lcohen@iyb.ca>; Comment <comment@noreply.github.com>
*Subject:* Re: [scakemyer/plugin.video.quasar] Quasar crashes with Android
7.0 (#544)



Wondering if bumping the API level wouldn't help here... After all we don't
support Kodi 15 anymore, and I think we could to drop support for Android 4
also.

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub
<https://github.com/scakemyer/plugin.video.quasar/issues/544#issuecomment-279523871>,
or mute the thread

**Comment by sask3m** [comment]
As far as the comment above I made about nexus player and Kodi 17 "unable to find quasar binary"  I fixed it by using the "arm" version of Kodi 17 and arm version of quasar.

**Comment by scakemyer** [comment]
That's never an excuse honestly. If you're stuck on Android 4, and stuck on Kodi 15, then you should be stuck on Quasar 0.9.x too.

**Comment by lcohen999** [comment]
No arguments from here, it was more a statement than a complaint (I have a
Shield/Mi/Asus) so I’m not worried there





*From:* scakemyer [mailto:notifications@github.com]
*Sent:* February 13, 2017 4:21 PM
*To:* scakemyer/plugin.video.quasar <plugin.video.quasar@noreply.github.com>
*Cc:* lcohen999 <lcohen@iyb.ca>; Comment <comment@noreply.github.com>
*Subject:* Re: [scakemyer/plugin.video.quasar] Quasar crashes with Android
7.0 (#544)



That's never an excuse honestly. If you're stuck on Android 4, and stuck on
Kodi 15, then you should be stuck on Quasar 0.9.x too.

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub
<https://github.com/scakemyer/plugin.video.quasar/issues/544#issuecomment-279525967>,
or mute the thread

**Comment by jhshadi** [comment]
@sask3m did you your solution works also for Nexus Player or is it exclusive to the Shield?

@scakemyer might be worth trying to upgrade the API, i'm willing to test it.

**Comment by sask3m** [comment]
@jhshadi     My fix for nexus player and kodi 17? I only tried it after downgrading to 6.0.

**Comment by jhshadi** [comment]
@sask3m right, so current working setup for Nexus player, Nougat (7.1.1) with Kodi 17 is still not found, correct?

**Comment by sask3m** [comment]
I never tried with 7.1 so can't say for sure. I had to use this fix with
6.0.

**Comment by jhshadi** [comment]
@sask3m and do you have Nougat (7.1.1) and Kodi 17 setup working with the Shield?

**Comment by sask3m** [comment]
Yes it works fine on my Shield. Both arm versions installed.

**Comment by compyy** [comment]
Sorry i dont understand, as per the logs it is running arm version, and not working, so you can see there is a issue.

**Comment by scakemyer** [comment]
@compyy That error has nothing to do with the issue at hand. See #714 and a few other ioerror-related issues that are all easily solved the same way, so please don't sidetrack this one here.

**Comment by compyy** [comment]
@scakemyer ok, i will try to remove the directory and see if it helps, thanks.

**Comment by jhshadi** [comment]
@scakemyer seems like @sask3m solution worked with the **Nexus Player** running Nougat (7.1.1) and Kodi 17.
After reading the logs i noticed it looks for the binary in the **arm** directory instead of the x86. i have installed the arm quasar release and it worked like a charm.

for some reason, after the Kodi 17 update, Kodi is recognizing the Nexus Player CPU architecture as ARM 32bit instead of x86 64bit.

all seems to work now. thanks for the awesome add-on.

**Comment by nsil60** [comment]
Hi all, im a bit confused by the above thread and how to get it working. I have the new Shield 2017 with Nougat 7.0 installed and NOT rooted. I installed the ARM version but getting the "unable to find quasar binary" message. I'm getting the same message on both SPMC Kodi ver 16.1 or Kodi 17. Can someone please let me know what i would need to do to get it working?

Thanks in advance.

**Comment by sask3m** [comment]
Kodi 17 under System/Settings/services/control there are two settings under "control" and both must be checked ON.  They are labeled  "allow remote control from applications" only one is ON by default.  Also you need to make sure that under system/settings you have settings level set at Expert so you able to view all the options.   The location of this may be in a different location using Spmc but its on there somewhere.

**Comment by nsil60** [comment]
@sask3m Thanks for the quick reply. Yes both are set as ON on both versions. One of your previous posts said you downgraded or are running on 6.0 Android on Nexus? So looks like we have different setups. Im looking for a solution for Shield/Nougat 7.0.

**Comment by sask3m** [comment]
I'm using a old Shield with no problems, are you sure you have the Android
arm Quasar and not linux arm by mistake? I'm not sure what to recommend,
grab a log file and submit I guess.

**Comment by nsil60** [comment]
@sask3m my apologies i downloaded the wrong version: plugin.video.quasar-0.9.74.android_x64.zip! I'll try the right version when i get a chance. Thanks a million.

**Comment by scakemyer** [comment]
I believe this can be closed, and #772 will take care of informing users which platform/arch package they should install.
